src/conversions.jl

```diff
@@ -134,6 +134,25 @@ function convert_arguments(::PointBased, positions::NTuple{N, AbstractVector}) w
     PlotSpec(points, tickranges = xyrange, ticklabels = labels)
 end
 
+function convert_arguments(
+        SL::SurfaceLike,
+        x::AbstractVector, y::AbstractVector, z::AbstractMatrix{<: Number}
```

Comment:
```suggestion
        x::AbstractVector, y::AbstractVector, z::AbstractMatrix

Reply:
Did you test the suggested change?

Reply:
Will test again, I don't remember to be honest